Library/Formula/nedit.rb

```diff
@@ -0,0 +1,39 @@
+class Nedit < Formula
+  homepage "http://www.nedit.org"
+  url "https://downloads.sourceforge.net/project/nedit/nedit-source/nedit-5.6-src.tar.gz"
+  sha1 "133641224b0969e31e3404b874c71c35341f028b"
+
+  depends_on :x11
+  depends_on "openmotif"
+
+  def install
+
+    inreplace "util/motif.c" do |s|
+      s.gsub! /200303/, "200304"
+    end
+
+    args = %W[
+      MOTIFLINK='-lXm'
+    ]
+
+    system "make", "macosx", *args
+    mv "source/nc", "source/ncl"
+
+    system "make -C doc man doc"
+    mv "doc/nedit.man", "doc/nedit.1"
+    mv "doc/nc.man", "doc/ncl.1"
+
+    bin.install "source/nedit"
+    bin.install "source/ncl"
+
+    man1.install "doc/nedit.1"
+    man1.install "doc/ncl.1"
+
+    doc.install Dir["doc/*"]
+  end
+
+  test do
+    system bin/"nedit", "-version"
+    system bin/"ncl", "-version"
```

Comment:
nedit is an X11 program, and I don't know how to write tests for it. Do you have a link to a suitable X11 brew that has acceptable tests?